src/components/create-safe/status/CreationStatus.tsx

```diff
@@ -1,154 +1,113 @@
-import { Box, Button, Divider, Grid, Paper, Typography } from '@mui/material'
-import { SafeCreationStatus, useSafeCreation } from '@/components/create-safe/status/useSafeCreation'
-import { useAppSelector } from '@/store'
-import { selectChainById } from '@/store/chainsSlice'
-import useChainId from '@/hooks/useChainId'
+import { Box, Button, Divider, Grid, Paper, Typography, Tooltip } from '@mui/material'
+import { SafeCreationStatus } from '@/components/create-safe/status/useSafeCreation'
 import EthHashInfo from '@/components/common/EthHashInfo'
 import Link from 'next/link'
 import { AppRoutes } from '@/config/routes'
 
-import css from './styles.module.css'
 import Track from '@/components/common/Track'
 import { CREATE_SAFE_EVENTS } from '@/services/analytics/events/createLoadSafe'
+import ComputedSafeAddress from '@/components/create-safe/status/ComputedSafeAddress'
+import useSafeCreationEffects from '@/components/create-safe/status/useSafeCreationEffects'
+import { useCallback } from 'react'
+import { useSafeCreation } from '@/components/create-safe/status/useSafeCreation'
+import type { StepRenderProps } from '@/components/tx/TxStepper/useTxStepper'
+import type { PendingSafeData } from '@/components/create-safe'
+import useLocalStorage from '@/services/local-storage/useLocalStorage'
+import StatusMessage from '@/components/create-safe/status/StatusMessage'
+import useWallet from '@/hooks/wallets/useWallet'
+import useIsWrongChain from '@/hooks/useIsWrongChain'
+import useStatus from '@/components/create-safe/status/useStatus'
+
+export const SAFE_PENDING_CREATION_STORAGE_KEY = 'pendingSafe'
 
 type Props = {
-  onClose: () => void
+  params: PendingSafeData
+  onSubmit: StepRenderProps['onSubmit']
+  onBack: StepRenderProps['onBack']
+  setStep: StepRenderProps['setStep']
 }
 
-const getStep = (status: SafeCreationStatus) => {
-  const loading = (
-    <img src="/images/open/safe-creation-process.gif" alt="Image of a vault that is loading" className={css.loading} />
+export const CreationStatus = ({ params, setStep }: Props) => {
+  const [status, setStatus] = useStatus()
```

Comment:
```suggestion
  const [status, setStatus] = useState<SafeCreationStatus>(SafeCreationStatus.AWAITING)
```

the `useStatus` hook is unnecessary

Reply:
I added it to make testing the component easier. Alternatively, we could also mock react useState.

Reply:
I think we should create a new hook if not needed. Should be possible to mock useState